sets/process/index_encode.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+from sets.core import MapStep
+
+
+class IndexEncode(MapStep):
```

Comment:
- Rename to `VocabularyEncode`.
- Allow to specify an existing vocabulary as list.
- Allow to get the current vocabulary as list.
- Shouldn't this inherit from `Embedding`?